src/hooks/tradingView/useTradingView.ts

```diff
@@ -70,14 +72,16 @@ export const useTradingView = ({
 
       tvWidgetRef.current.onChartReady(() => {
         tvWidgetRef?.current?.headerReady().then(() => {
-          displayButtonRef.current = tvWidgetRef?.current?.createButton();
-          displayButtonRef.current.innerHTML = `<span>${stringGetter({
-            key: STRING_KEYS.ORDER_LINES,
-          })}</span> <div class="displayOrdersButton-toggle"></div>`;
-          displayButtonRef.current.setAttribute(
-            'title',
-            stringGetter({ key: STRING_KEYS.ORDER_LINES_TOOLTIP })
-          );
+          if (displayButtonRef) {
+            displayButtonRef.current = tvWidgetRef?.current?.createButton();
+            displayButtonRef.current.innerHTML = `<span>${stringGetter({
+              key: STRING_KEYS.ORDER_LINES,
+            })}</span> <div class="displayOrdersButton-toggle"></div>`;
+            displayButtonRef.current.setAttribute(
+              'title',
+              stringGetter({ key: STRING_KEYS.ORDER_LINES_TOOLTIP })
+            );
+          }
```

Comment:
double check type errors on setting the ref.
1. expecting `null` instead of `undefined`
2. current._ may be `undefined`

Reply:
Also double check line 70. TypeError on `options`